front/src/components/student-registration/student-registration-form/student-registration-form.js

```diff
@@ -0,0 +1,203 @@
+import React, { Component } from 'react';
+import { studentRegistered } from '../../../common/redux/actions';
+import { Row, Form, Typography } from 'antd';
+import { Col, Input, Button } from '../student-registration-form/style';
+
+const { Title } = Typography;
+
+const layout = {
+  labelCol: { span: 8 },
+  wrapperCol: { span: 16 },
+};
+
+const tailLayout = {
+  wrapperCol: {
+    offset: 9,
+    span: 16,
+  },
+};
+
+class StudentRegistrationPage extends Component {
+  constructor(props) {
+    super(props);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.validateEng = this.validateEng.bind(this);
+    this.validateUkr = this.validateUkr.bind(this);
+    this.validatePhoneNumber = this.validatePhoneNumber.bind(this);
```

Comment:
Do we need to bind all of these methods, if they don't use `this`

Reply:
Done